conda/conda-build/conda_build_config.yaml

```diff
@@ -1,3 +1,6 @@
+gpu_enabled:
+  - true
+  - 
```

Comment:
Should we leave a comment that this corresponds to false? It's pretty mystifying otherwise. :)

Reply:
I guess that would be the same comment for legate.core.

Reply:
Updated